### DISCUSSION TITLE
安卓设置字重不可行

### DISCUSSION THREAD

**Discussion by 19snow93** [Tencent/libpag#2107] [Q&A] [answered]
![image](https://github.com/Tencent/libpag/assets/18436794/41fb8a53-7c8a-47c0-82d7-30110945e25b)
设置了字重没有生效是什么问题，请大神们指教

**Comment by kevingpqi123** [comment, marked as answer]
设置方法是错误的，这些信息都是需要和字体去匹配的，如果没有匹配上就是无效的，参考下下面的用法
<img width="900" alt="image" src="https://github.com/Tencent/libpag/assets/16609221/2e3371c9-0790-4fd9-9bd8-ab6966ba991b">

**Reply by 19snow93** [reply]
请问这个字体都要在哪里下载，我自己下载了几个，都解析不出来

**Reply by kevingpqi123** [reply]
字体一般都需要版权，找你们的设计师
